src/kafka/clusterID

```diff
@@ -0,0 +1 @@
+ckjPoprWQzOf0-FuNkGfFQ
```

Comment:
Is this file required?

If so, please just add a new line at the end of the file.

Reply:
The confluent Kafka image requires this file and fails if it has an empty line.